contrib/pyln-testing/pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
+name = "pyln-testing"
+version = "0.10.2"
+description = "Test your c-lightning integration, plugins or whatever you want"
+authors = ["Christian Decker <decker.christian@gmail.com>"]
+license = "BSD-MIT"
+
+packages = [
+  { include = "pyln/testing" },
+]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+pytest = "^7.0.1"
```

Comment:
maybe dev dependencies for pytest is a better place?

Reply:
Hm, good question. This package is a bit special in that it's going to be a dev-dependency for others most likely.

The way I'm thinking about this is that the package is not functional without `pytest` so it's not a dev-dependency, but rather a normal dependency.

Reply:
Sorry for the latency to add the comments here at different time.

>The way I'm thinking about this is that the package is not functional without pytest so it's not a dev-dependency, but rather a normal dependency.

I got it, and I think it is safe this put in the not dev dependencies because it help also to resolve the conflict in some configurations when an older version of `pytest` is used. Sorry if I miss this though during the review